### ISSUE TITLE
Compile error of Aseba

### ISSUE THREAD

**Issue by WafaJohal** [aseba-community/thymio-tracker#1]
I am trying to compile the demo (following [](https://github.com/aseba-community/thymio-tracker/blob/demo/DEMO.md))
But the build fails when I run the android.sh command it fails at 97% on the compile of  
`CMakeFiles/thymiotracker.dir/aseba/compiler/tree-optimize.cpp.o`
The first error I get is a weird error: 
`thymio-tracker/aseba/common/utils/utils.h:63:7: error: 'unique_ptr' in namespace 'std' does not name a template type  std::unique_ptr<T> make_unique(Args&&... args)`

Is there a particular version of g++ to be used to compile aseba?

**Comment by stephanemagnenat** [comment]
> Is there a particular version of g++ to be used to compile aseba?

Yes, because of C++11/14 version support. This version is checked in the main `CMakeLists.txt` of Aseba but of course this check does not happen when used with submodules. Recently we switched Aseba to C++14. To answer your question one needs to know more about the setup you are using, and which `android.sh` you refer to.
